DagsFactory/templates/prod_template.py

```diff
@@ -0,0 +1,15 @@
+import pendulum
+from Utils.ParametersLoader import load_configs
+from airflow import DAG
+from airflow.providers.google.cloud.operators.dataproc import DataprocSubmitSparkJobOperator
+
+with DAG(dag_id='$dag_id', start_date=pendulum.parse('2020/01/01'), schedule_interval=$schedule_interval) as dag:
+    configs = load_configs('$configs_filepath')
+
+    run_job = DataprocSubmitSparkJobOperator(
```

Comment:
I assume u will use a temp cluster (not permanent). That's why u forgot about 2 tasks, create and delete a cluster